x/paramHub/hub.go

```diff
@@ -83,6 +83,16 @@ func RegisterUpgradeBeginBlocker(paramHub *ParamHub) {
 		}
 		paramHub.UpdateFeeParams(ctx, crossStakeFeeParams)
 	})
+	sdk.UpgradeMgr.RegisterBeginBlocker(sdk.BEP159, func(ctx sdk.Context) {
+		updateFeeParams := []param.FeeParam{
+			&param.FixedFeeParams{MsgType: "create_validator_open", Fee: CreateValidatorFee, FeeFor: sdk.FeeForProposer},
```

Comment:
Because previously `MsgCreateValidatorProposal` doesn't define its own `Type()` method, it inherits the function from `MsgCreateValidator`, which takes `create_validator` already.